skyportal/handlers/api/source.py

```diff
@@ -692,14 +692,19 @@ def post_source(data, user_id, session, refresh_source=True):
         raise AttributeError(
             "You must belong to one or more groups before you can add sources."
         )
-    try:
-        group_ids = [
-            int(id)
-            for id in data.pop('group_ids')
-            if int(id) in user_accessible_group_ids
-        ]
-    except KeyError:
+
+    group_ids = data.pop('group_ids')
+    if group_ids is None:
         group_ids = user_group_ids
+    else:
+        group_ids = []
```

Comment:
don't think that one works 😉 @mcoughlin

Reply:
you overwrite it and loop over it, but well it will always be empty

Reply:
Thanks bud! Should be fixed.